### ISSUE TITLE
Keyboard Access > Failing Example 2 > Sample page not keyboard accessible on Firefox

### ISSUE THREAD

**Issue by JRNPW** [Section508Coordinators/DHSA-TT-200-A#360]
## Issue Type (put a capital X between the brackets to indicate issue type)
- [X] System Bug (the course doesn't function properly)
- [ ] Typo/grammar
- [ ] Incorrect content
- [ ] 508 issue (something is not accessible according to our own test process)
- [ ] Other, specify...[replace this text with your issue type description]

## Description
On Example 2 of "Keyboard Access Failing Examples," the website which is linked cannot be accessed on Firefox via the keyboard.

## Location
Topic 4: Keyboard Access and Focus > Lesson 1: Keyboard Access > Failing Examples > Example 2 (https://training.section508testing.net/mod/book/view.php?id=1229&chapterid=2676)

Sample website: https://dev.section508testing.net/html/4A_Fail2.html

## Actual Behavior / Observation
ANDI indicates that various elements are focusable, but using the TAB key, a Firefox user cannot get to ANY of the buttons or links (not even the ones that the text of the Lesson indicates should be reachable).

## Expected Behavior / Recommendation
Firefox users should also be able to access the elements indicated in the Lesson text.

**Comment by drewnielson** [comment]
@JRNPW this obviously seems related to issue #361 that you also noted. Would you mind confirming in this case too that it is indeed an keyboard access issue and not a visible focus issue. Just haven't been able to replicate with my version of Firefox.

**Comment by JRNPW** [comment]
Sorry for the delay, @drewnielson; other work priorities outranked my training for the last several days.

Yes, this definitely still happens (as of a few moments ago) on Firefox version 70 on macOS Mojave. Fwiw, I've had the same issue with a web app that we're developing: Elements on a page that are keyboard-accessible in Chrome are NOT accessible on Firefox (and also on Safari). I've raised bug reports, but they haven't been picked up by the Dev Team yet.

**Comment by drewnielson** [comment]
No problem. Thanks for following up. This will definitely require some continued investigation.

**Comment by JRNPW** [comment]
I suspect that this is related to the settings mentioned in this [Mozilla Developers Network page](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Cross_browser_testing/Accessibility). Therefore, probably not a bug but a misconfiguration of the Firefox browser.
